lib/support/windows.js

```diff
@@ -0,0 +1,84 @@
+const os = require('os');
+const fs = require('fs');
+const path = require('path');
+const child_process = require('child_process');
+const utils = require('../utils.js');
+const KiteError = require('../kite-error');
+
+const {STATES} = require('../constants');
+
+if (os.platform() !== 'win32') {
+  process.env.TMP = os.tmpDir();
+  process.env.ProgramW6432 = os.tmpDir();
+  process.env.LOCALAPPDATA = os.tmpDir();
+}
+
+const WindowsSupport = {
+  RELEASE_URL: 'https://alpha.kite.com/release/dls/windows/current',
+  KITE_INSTALLER_PATH: path.join(process.env.TMP, 'Kite.exe'),
+  KITE_EXE_PATH: path.join(process.env.ProgramW6432, 'Kite', 'kited.exe'),
```

Comment:
I'll need to find a way to read a registry key from node, but that aside I don't see any issue with that.